core/controllers/improvements.py

```diff
@@ -35,10 +35,11 @@ def get_task_dict_with_username_and_profile_picture(task_entry):
     URL inserted.
 
     Args:
-        task_entry: improvements_domain.TaskEntry. The TaskEntry domain object.
+        task_entry: improvements_domain.TaskEntry. The TaskEntry domain object
+            whose dict is to be returned.
 
     Returns:
-        TaskEntryDict. TaskEntry dict username and profile picture
+        TaskEntryDict. TaskEntry dict with username and profile picture
```

Comment:
I think you'd need to say "resolver" somewhere here, otherwise it's not clear whose username and profile picture you're talking about.

Reply:
Done.